src/lookupStream.js

```diff
@@ -31,11 +31,18 @@ function createPipResolverStream(pipResolver) {
     }
 
     pipResolver.lookup(doc.getCentroid(), getAdminLayers(doc.getLayer()), (err, result) => {
-
-      // assume errors at this point are fatal, so pass them upstream to kill stream
       if (err) {
-        logger.error(err);
-        return callback(new Error('PIP server failed: ' + (err.message || JSON.stringify(err))));
+        // if there's an error, just log it and move on
+        const parts = [
+          `id:${doc.getGid()}`,
+          `lat:${doc.getCentroid().lat}`,
+          `lon:${doc.getCentroid().lon}`,
+          `PIP server failed: ${(err.message || JSON.stringify(err))}`
+        ];
+
+        logger.error(parts.join('|'));
```

Comment:
according to [this doc](https://github.com/winstonjs/winston#logging-with-metadata), you can do something like 

```javascript
logger.error(`PIP server failed: ${err.message || JSON.stringify(err)}`, 
{
  id: doc.getGid(),
  lat: doc.getCentroid().lat, 
  lon: doc.getCentroid().lon
});
```